airflow/www/templates/airflow/dag.html

```diff
@@ -125,12 +125,12 @@ <h4 class="pull-right">
       <div class="modal-content">
         <div class="modal-header">
           <h4 class="modal-title" id="myModalLabel">
-            <span id='task_id'></span>
-            <button id="btn_filter" type="button" class="btn btn-default" title="Filter on this task and upstream ">
+            <span id="task_id"></span>
+            <a id="btn_filter" type="button" class="btn btn-default" title="Filter on this task and upstream ">
```

Comment:
`<a type=button>` is not a valid property :)

Reply:
Thanks, updated to drop `type="button"` from all anchors in this template.